### PR TITLE
Refactored the code to use tabcmd instead of tabcmd2

### PR DESCRIPTION
**Pull request by ovinis**

### CODE REVIEW

**Comment by jacalata** [comment]
Is it worth changing the directory name from pythontabcmd to just tabcmd? I think labelling it python is unnecessary, since the sibling test folder and setup.py are all python anyway - if we add a client in another language we'd probably want to introduce a new layer above this.

**Comment by ovinis** [comment]
yes, I think its unnecessary too. We'd want to have it open for any future clients. Thanks for catching that.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 [![Tableau Supported](https://img.shields.io/badge/Support%20Level-Tableau%20Supported-53bd92.svg)](https://www.tableau.com/support-levels-it-and-developer-tools)
 
 A Python based app that replicates the functionality of the existing [Tabcmd command line utility](https://help.tableau.com/current/server/en-us/tabcmd.htm).
 
-**Important Note:** Tabcmd2 is a work in progress ("beta") which may be useful for test and development purposes, but is not yet recommended for production environments.
+**Important Note:** tabcmd is a work in progress ("beta") which may be useful for test and development purposes, but is not yet recommended for production environments.
 
-* [Why Tabcmd2\?](#whytabcmd2)
+* [Why a Python based tabcmd\?](#whytabcmd)
```
Comment (jacalata) [inline README.md:9]: would make more sense as "Why this new version of tabcmd?"

File: tabcmd/parsers/help_parser.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import sys
+import argparse
+
+
+class HelpParser:
+    @classmethod
+    def print_help_description(cls):
+        description = \
+            "tabcmd - Tableau Server Command Line Utility 2.0 \n \n" \
```
Comment (jacalata) [inline tabcmd/parsers/help_parser.py:9]: shouldn't argparse generate this for us?

**Review by jacalata — APPROVED** [review]
looks good!

File: tabcmd/parsers/help_parser.py (hunk 2 above)
Comment (ovinis) [reply tabcmd/parsers/help_parser.py:9]: Thanks for reviewing. 
I had to override the Tabcmd 2 argparse help message to to mirror the same help message given out in Tabcmd1.

**Pull request merged**